### PR TITLE
PSA 1.0 beta3 minor fixes

### PR DESCRIPTION
**Pull request by gilles-peskine-arm**
Minor fixes discovered while proofreading beta3 rc3, to be inclused in beta3 rc4.

### CODE REVIEW

File: include/psa/crypto_sizes.h [hunk 1]
```diff
@@ -641,14 +641,16 @@
...
- * status = psa_get_key_information(key, &key_type, &key_bits);
+ * status = psa_get_key_attributes(key, &attributes);
  * if (status != PSA_SUCCESS) handle_error(...);
+ * psa_key_type_t key_type = psa_get_key_type(&attributes);
+ * size_t key_bits = psa_get_key_bits(&attributes);
  * size_t buffer_size = PSA_KEY_EXPORT_MAX_SIZE(key_type, key_bits);
+ * psa_reset_key_attributes(&attributes);
  * unsigned char *buffer = malloc(buffer_size);
- * if (buffer != NULL) handle_error(...);
+ * if (buffer == NULL) handle_error(...);
```
Comment (Patater) [inline include/psa/crypto_sizes.h]: lol. I love untested code samples.

File: include/psa/crypto.h [hunk 2]
```diff
@@ -461,10 +461,17 @@ void psa_reset_key_attributes(psa_key_attributes_t *attributes);
...
- * The handle should eventually be closed with psa_close_key()
- * to release associated resources.
+ * Open a handle to a persistent key. A key is persistent if it was created
+ * with a lifetime other than #PSA_KEY_LIFETIME_VOLATILE. A persistent key
+ * always has a nonzero key identifier, set with psa_set_key_id() when
+ * creating the key. Implementations may provide additional pre-provisioned
+ * keys with identifiers in the range
+ * #PSA_KEY_ID_VENDOR_MIN&ndash;#PSA_KEY_ID_VENDOR_MAX.
+ *
+ * The application must eventually be close the handle with psa_close_key()
```
Comment (Patater) [inline include/psa/crypto.h]:
```diff
- The application must eventually be close
+ The application must eventually close
```

File: include/psa/crypto_values.h [hunk 3]
```diff
@@ -373,7 +373,7 @@
  */
 #define PSA_KEY_TYPE_DERIVE                     ((psa_key_type_t)0x52000000)
 
-/** Key for an cipher, AEAD or MAC algorithm based on the AES block cipher.
+/** Key for a cipher, AEAD or MAC algorithm based on the AES block cipher.
```
Comment (Patater) [inline include/psa/crypto_values.h]: Commit message "Grammar" should be "Fix grammar"

File: include/psa/crypto.h (hunk 2 above)
Comment (gilles-peskine-arm) [reply include/psa/crypto.h]: Fixed by amending the commit.

File: include/psa/crypto_values.h (hunk 3 above)
Comment (gilles-peskine-arm) [reply include/psa/crypto_values.h]: I don't see why, but ok, whatever.

Comment (Patater) [reply include/psa/crypto_values.h]: Gotta start with the verb, man.

**Comment by Patater** [comment]
Manually tested by @gilles-peskine-arm with Doxygen and clang. Doc-only changes, so no need to run full CI.

**Pull request merged**